app/src/main/res/layout/screen_patient_facility_change.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can we squash the commit with a later commit (`Add UI functions to PatientFacilityChangeActivity`) since this change seems to have been reverted in that commit.

Reply:
Done